presto-parser/src/main/java/com/facebook/presto/sql/tree/AtTimeZone.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.tree;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+public class AtTimeZone
+        extends Expression
+{
+    private final Expression valueExpression;
+    private final Expression timeZoneSpecifier;
+
+    public static final QualifiedName FUNCTION_NAME = QualifiedName.of("at_timezone");
```

Comment:
This shouldn't be in the AST. It's an internal detail of the way the planner implements this operation.